RELEASING.md

```diff
@@ -0,0 +1,36 @@
+# Releasing Guide
+
+AX releasing workflow is semi-guided by the CI logic comprised of [GitHub actions scripts](.github/workflows/), [makefile](./Makefile), and [cosmos-release tool](./rust/release/).
```

Comment:
“comprised of” is almost always incorrect: to comprise means to “take together”, i.e. to gather, so “comprised of” would mean “gathered by”.
```suggestion
The AX releasing workflow is semi-guided by the CI logic comprising [GitHub actions scripts](.github/workflows/), [makefile](./Makefile), and [cosmos-release tool](./rust/release/).
```